server/server.js

```diff
@@ -42,9 +43,14 @@ app.post('/ratingsForEntireStreet', (req, res) => {
 
 app.post('/ratings', (req, res) => {
   let directions = req.body.streets;
+  
   utils.findCrimesByLine(directions)
     .then((crimesPerStreet) => {
-      res.send(JSON.stringify(crimesPerStreet));
+      let crimeData = {
+        crimesPerStreet: crimesPerStreet,
+        ratingInfo: ratingInfo,
```

Comment:
Might make more sense to separate out ratingInfo into it's own post request that the client can query upon initialization (rather than providing the same data every single time new directions are queried)